DevCycle.SDK.Server.Common/Model/Cloud/Variable.cs

```diff
@@ -64,28 +67,26 @@ public Variable(string key, T defaultValue, string evalReason)
         {
 
         }
-        
-        /// <summary>
-        /// unique database id
-        /// </summary>
-        /// <value>unique database id</value>
-        [DataMember(Name="_id", EmitDefaultValue=false)]
-        public string Id { get; set; }
 
         /// <summary>
         /// Unique key by Project, can be used in the SDK / API to reference by &#x27;key&#x27; rather than _id.
         /// </summary>
         /// <value>Unique key by Project, can be used in the SDK / API to reference by &#x27;key&#x27; rather than _id.</value>
-        [DataMember(Name="key", EmitDefaultValue=false)]
+        [DataMember(Name="key")]
         public string Key { get; set; }
 
 
         /// <summary>
         /// Variable value can be a string, number, boolean, or JSON
         /// </summary>
         /// <value>Variable value can be a string, number, boolean, or JSON</value>
-        [DataMember(Name="value", EmitDefaultValue=false)]
+        [DataMember(Name="value")]
         public T Value { get; set; }
```

Comment:
Now that I think of it - `T` will actually become `JObject` or `JArray` - which we should check for how to handle JSON types.

Reply:
i think it parses it as the type that is expected given the type T from the default value

Reply:
I'm more thinking from the JSON object representation - which might mess up the `Type` prop. 

Though - it's just something we'd have to test on the example app with a JSON variable.

Reply:
default values also can't be arrays, that's not a valid type for a variable

Reply:
It is now that you've set the default value to `T`. `JObject` is from Newtonsoft - which is how JSON variables get deserialized into C# - which is gonna add a bit of confusion on type enums.